### DISCUSSION TITLE
Arm builds

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#559] [General]
Has anyone provided any arm builds of audiogridder or will they be provided in future releases ??

I feel this app will be very good for these types of computers because arm computers don’t have the plugin library that windows and Mac OS have

**Comment by apohl79** [comment]
https://audiogridder.com/download/ ;-)

**Reply by bsjdsp** [reply]
I don’t see any arm builds

**Reply by apohl79** [reply]
just look at the apple arm64 pkg files.

**Reply by bsjdsp** [reply]
But aren’t those for the Apple m1. I’m looking for a package to work with the raspberry Pi 4

**Reply by apohl79** [reply]
Ok I see, you didn't say that. No there is no raspberry builds.

**Reply by bsjdsp** [reply]
I understand the confusion because raspberry Pi is ARM based just like Apple M1 is but I assume the way it’s built is different. It’s still possible to compile for other ARM based computers such as raspberry Pi etc. I’m really looking forward to using this for the Pi when I get one
